benchmarks/synchrobench/run.sh

```diff
@@ -3,10 +3,10 @@
 dir=`pwd`
 output=${dir}/output
 java=java
-jarfile="target/oak-benchmarks-synchrobench-0.1.4-SNAPSHOT.jar"
+jarfile="target/oak-benchmarks-synchrobench-0.1.6-SNAPSHOT.jar"
 
 thread="01 04 08 12"
-size="5000000"
+size="10000000"
```

Comment:
To change the path to jar is OK. But to change other parameters I am not sure... One one hand, anyway everyone should actually change it. One the other hand, our previous "regression" was without this change.

Reply:
The size used in the paper is 10M. This way the right execution command is created.

Reply:
OK